theano/gpuarray/tests/test_elemwise.py

```diff
@@ -52,6 +54,85 @@ def test_elemwise_pow():
+            self.expected_erfcinv_outputs[dtype] = scipy.special.erfcinv(numpy_array)
+
+    def check_gpu_scalar_op(self, theano_function, scalar_optype):
+        for node in theano_function.maker.fgraph.apply_nodes:
+            if isinstance(node.op, GpuElemwise) and isinstance(node.op.scalar_op, scalar_optype):
+                return True
+        theano.printing.debugprint(theano_function)
+        return False
+
+    def compute_erfinv_host(self, dtype):
+        vector = theano.tensor.vector(dtype=dtype)
+        output = theano.tensor.erfinv(vector)
+        f = theano.function([vector], output, name='HOST/erfinv/' + dtype, mode=mode_without_gpu)
+        assert len([n for n in f.maker.fgraph.apply_nodes if isinstance(n.op, GpuElemwise)]) == 0
+        vector_val = self.default_arrays[dtype]
+        f(vector_val)
+        out = f(vector_val)
+        assert_allclose(self.expected_erfinv_outputs[dtype], out)
+
+    def compute_erfinv_gpu(self, dtype):
+        vector = theano.tensor.vector(dtype=dtype)
+        output = theano.tensor.erfinv(vector)
+        f = theano.function([vector], output, name='GPU/erfinv/' + dtype, mode=mode_with_gpu)
+        if not theano.config.device.startswith('opencl'):
+            assert self.check_gpu_scalar_op(f, GpuErfinv), 'Function graph does not contains scalar op "GpuErfinv".'
+        vector_val = self.default_arrays[dtype]
+        f(vector_val)
+        out = f(vector_val)
+        assert_allclose(self.expected_erfinv_outputs[dtype], out)
+
+    def compute_erfcinv_host(self, dtype):
+        vector = theano.tensor.vector(dtype=dtype)
+        output = theano.tensor.erfcinv(vector)
+        f = theano.function([vector], output, name='HOST/erfcinv/' + dtype, mode=mode_without_gpu)
+        assert len([n for n in f.maker.fgraph.apply_nodes if isinstance(n.op, GpuElemwise)]) == 0
+        vector_val = self.default_arrays[dtype]
+        f(vector_val)
+        out = f(vector_val)
+        assert_allclose(self.expected_erfcinv_outputs[dtype], out)
+
+    def compute_erfcinv_gpu(self, dtype):
+        vector = theano.tensor.vector(dtype=dtype)
+        output = theano.tensor.erfcinv(vector)
+        f = theano.function([vector], output, name='GPU/erfcinv/' + dtype, mode=mode_with_gpu)
+        if not theano.config.device.startswith('opencl'):
+            assert self.check_gpu_scalar_op(f, GpuErfcinv), 'Function graph does not contains scalar op "GpuErfcinv".'
+        vector_val = self.default_arrays[dtype]
+        f(vector_val)
+        out = f(vector_val)
+        assert_allclose(self.expected_erfcinv_outputs[dtype], out)
+
+    def test_elemwise_erfinv(self):
+        for dtype in self.dtypes:
+            self.compute_erfinv_host(dtype)
+            self.compute_erfinv_gpu(dtype)
+
+    def test_elemwise_erfcinv(self):
+        for dtype in self.dtypes:
+            self.compute_erfcinv_host(dtype)
+            self.compute_erfcinv_gpu(dtype)
```

Comment:
If you use the functions compute_* at only 1 places and they are short, why did you put them in functions? This make the code longer and more complicated to understand and cause some extra duplicate code.

But otherwise, it is good.